core/app/models/spree/payment/processing.rb

```diff
@@ -40,7 +40,12 @@ def capture!(amount=nil)
 
           money = ::Money.new(amount, currency)
           capture_events.create!(amount: money.to_f)
-          handle_response(response, :complete, :failure)
+
+          if uncaptured_amount > 0
+            handle_response(response, :pend, :failure)
+          else
+            handle_response(response, :complete, :failure)
+          end
```

Comment:
Good call.  I think what I'm actually going to do though is leave the current implementation, and rather than doing partial captures on a single payment to split the captured amount into it's own new payment record.  The CaptureEvent model keeping track of the captured amount introduces a can of worms @jawthorn brought up on a commit I reverted and stuck into this PR that I think simply breaking into multiple payments would solve.